docs/reference/platforms/android/gradle.rst

```diff
@@ -102,6 +102,22 @@ You can specify a background color for the splash screen using the
 
 Android projects do not support installer images.
 
+Colors
+======
+
+In addition to ``splash_background_color``, Android projects allow for
+configuration of an ``accent_color``, ``primary_color``, and
+``primary_color_dark``.
+
+* ``accent_color`` is used subtly throughout an Android app to call attention
+  to key elements. It's used on things like form labels and inputs.
```

Comment:
```suggestion
* ``accent_color`` is used as a subtle highlight throughout your app to call attention
  to key elements. It's used on things like form labels and inputs.
```

Reply:
I made all those changes. Thanks for helping me get this in shape!